Cargo.toml

```diff
@@ -3,11 +3,11 @@ name = "template-benchmarks-rs"
 version = "0.1.0"
 authors = ["Dirkjan Ochtman <dirkjan@ochtman.nl>"]
 build = "src/build.rs"
-edition = "2018"
+edition = "2021"
 
 [dependencies]
 askama = { git = "https://github.com/djc/askama", branch = "main" }
-criterion = "0.4"
+criterion = { version = "0.4", features = ["html_reports"] }
```

Comment:
Why enable the HTML reports here?

Reply:
Added because it was not generating the svg output even if _(gnuplot was installed)_.